src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,301 @@
+package duke;
+
+import duke.exception.AddDeadlineException;
+import duke.exception.AddEventException;
+import duke.exception.EmptyNameException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void main(String[] args) {
+        loadFromDisk();
+        printWelcomeMsg();
+        waitForCommand();
```

Comment:
method name could be better. waitForCommand gives the impression that the function does not do any processing.

Reply:
Okay, I'll see what I can do. Thanks!